update-regression-tests-gitlab

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
+# This script is solely used on gitlab
+
+GITHUB=github.com
+
+# getting tests
+git clone https://gitlab-ci-token:${CI_JOB_TOKEN}@${GITHUB}/pulp-platform/regression_tests.git
```

Comment:
Comment on cloning a specific version also applies here.

Why the complex cloning URL?  Does `https://github.com/pulp-platform/regression_tests.git` not work?

Reply:
I though that gitlab_ci needed a particular sintax with the token. It works with the simple URL too.